docs/sla.md

```diff
@@ -1,3 +1,3 @@
-# Meltano Cloud Service Level Agreement (SLA)
+# Meltano Cloud Platform Service Level Agreement (SLA)
 
-Todo
+# Meltano Cloud Platinum Connector Service Level Agreement (SLA)
```

Comment:
This is the main SLA page we'll link from the contract, _not_ specific to Platinum Connector Support, so please roll back this change!